### PR TITLE
Implement a simple health check

### LINKED ISSUE #179: Health check for integration into monitoring
Hi would it be possible to add a health check to logspout?

e.g. curl http://localhost:8000/health

Returns 200 OK if the process is working otherwise an error

Want to integrate it into a Consul Healthcheck

### PR DESCRIPTION
**Pull request by vbeausoleil**
I am currently using Logspout on a DC/OS cluster and I needed to have a health check route in order for the health check to succeed.

Health check route is /health and returns the string "Healthy"

This PR might fix #179

### CODE REVIEW

File: router/http.go [hunk 1]
```diff
@@ -31,4 +31,4 @@ func (s *httpService) Setup() error {
 
 func (s *httpService) Run() error {
 	return http.ListenAndServe(":"+s.port, nil)
-}
+}
```
Comment (michaelshobbs) [inline router/http.go]: missing newline at EOF

File: routesapi/routesapi.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ func init() {
 func RoutesAPI() http.Handler {
 	routes := router.Routes
 	r := mux.NewRouter()
-
+
```
Comment (michaelshobbs) [inline routesapi/routesapi.go]: superfluous whitespace

File: healthcheck/healthcheck.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+// HealthCheck returns a http.Handler for the health check
+func HealthCheck() http.Handler {
+	r := mux.NewRouter()
+	r.HandleFunc("/health", func(w http.ResponseWriter, req *http.Request) {
+		w.Write([]byte("Healthy!\n"))
+	})
+	return r
+}
```
Comment (michaelshobbs) [inline healthcheck/healthcheck.go]: missing newline at EOF

**Comment by vbeausoleil** [comment]
@michaelshobbs: I just implemented the requested changes. Let me know if something needs to be adjusted.

I was reminded about this PR by @henkjan

**Pull request merged**